### ISSUE TITLE
Phone input field

### ISSUE THREAD

**Issue by Blackbaud-ToddRoberts** [blackbaud/skyux1#583]
Bring 3rd party phone input field into SkyUX 
http://jackocnr.com/intl-tel-input.html

**Needs modification to meet accessibility standards**
1.  Lack of labels – there a actually two controls here so would need labels for both (country and phone number). Imagine this could be added when implemented.
2a. The country dropdown is missing semantics completely for a screenreader. It’s set up as a div and a unordered list instead of a form input. Semantically it should be either be select/option elements or be given new semantics and aria states. The div element would need role=“listbox” and an additional state for child selected and its children need to have the roles of “option”. More info here - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_listbox_role
2b. Voiceover also doesn’t seem to pick up on the element though it’s in the tab index though thinking it could be related to not having the semantics correct.

**Needs styling tweaks to mirror other SkyUX input fields**
• Increasing the size of the input
• Line colors, corner radii, remove inner shadow
• Font family, size, color
• Use a FontAwesome caret
• onFocus and failed validation styles
• Increasing height of the “rows” in the drop menu
• Adding onTouch/onMouseDown styles to the “rows”
• No inertial scrolling inside the drop menu – can we add that?

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.6.3](https://github.com/blackbaud/skyux/releases/tag/v1.6.3)
